README.md

```diff
@@ -26,8 +25,8 @@ Please note Baritone is no longer included. Download the standalone jar [from he
 
 ## Installing
 
-KAMI is a forge mod. Start by downloading the latest version of [1.12.2 forge](https://files.minecraftforge.net/).
-1. Install forge
+The `master` branch of KAMI is a 1.12.2 Forge mod. Start by downloading the latest version of [1.12.2 Forge](https://files.minecraftforge.net/).
```

Comment:
This sounds weird. Just say that the 1.12.2 version of KAMI is a forge mod.

Also, please point the link to https://files.minecraftforge.net/maven/net/minecraftforge/forge/index_1.12.2.html